### ISSUE TITLE
app does not play and File does not exist: owncloud/apps/music/js/vendor/underscore/underscore-min.map

### ISSUE THREAD

**Issue by Wikinaut** [owncloud/music#426]
I wanted to try the music app, but it does not work for me.

The App scanned and found some mp3 and ogg music files, but when trying to click, it does not play music. The cursors jumps on the screen. No player module is shown.
- OC server 8.1.1
- music app bd492a38b0f90004805f0f80547d546aa2f7fa7d
- Client: Firefox 40.0.3 on openSUSE Tumbleweed (this FF plays mp3 files, I double-checked this).

I have no idea, whether this is related but my Server log file shows 

```
File does not exist: owncloud/apps/music/js/vendor/underscore/underscore-min.map
```

**Comment by MorrisJobke** [comment]
> File does not exist: owncloud/apps/music/js/vendor/underscore/underscore-min.map

This is unrelated and only for debugging needed.

When you hover on a song/album/artist the play icon should be shown. Is this the case? Then you can click on it and it should start to play or show a notification, that the music file is not supported.

**Comment by Wikinaut** [comment]
music app does not work ( https://github.com/owncloud/music/commit/e5f1d52a5aac617974001b5723afb762044d344f ) and results in such error log lines when clicking on a music file. Pls. can you assist.

{"reqId":"HbZULHmUx4DavCdIJXaP","remoteAddr":"1.2.3.4","app":"index","message":"Exception: {\"Exception\":\"OCP\\\\Files\\\\NotFoundException\",\"Message\":\"\",\"Code\":0,\"Trace\":\"#0\\\ /srv/owncloud\\\/apps\\\/music\\\/controller\\\/apicontroller.php(116): OCA\\\\Music\\\\Db\\\\Track->toCollection(Object(OC\\\\URLGenerator), Object(OC\\\\Files\\\\Node\\\\Folder))\\n#1 [internal function]: OCA\\\\Music\\\\Controller\\\\ApiController->collection()\\n#2 \\\/srv/owncloud\\\/lib\\\/private\\\/appframework\\\/http\\\/dispatcher.php(159): call_user_func_array(Array, Array)\\n#3 \\\/srv/owncloud\\\/lib\\\/private\\\/appframework\\\/http\\\/dispatcher.php(89): OC\\\\AppFramework\\\\Http\\\\Dispatcher->executeController(Object(OCA\\\\Music\\\\Controller\\\\ApiController), 'collection')\\n#4 \\\/srv/owncloud\\\/lib\\\/private\\\/appframework\\\/app.php(108): OC\\\\AppFramework\\\\Http\\\\Dispatcher->dispatch(Object(OCA\\\\Music\\\\Controller\\\\ApiController), 'collection')\\n#5 \\\/srv/owncloud\\\/lib\\\/private\\\/appframework\\\/routing\\\/routeactionhandler.php(45): OC\\\\AppFramework\\\\App::main('ApiController', 'collection', Object(OC\\\\AppFramework\\\\DependencyInjection\\\\DIContainer), Array)\\n#6 [internal function]: OC\\\\AppFramework\\\\routing\\\\RouteActionHandler->__invoke(Array)\\n#7 \\\/srv/owncloud\\\/lib\\\/private\\\/route\\\/router.php(274): call_user_func(Object(OC\\\\AppFramework\\\\routing\\\\RouteActionHandler), Array)\\n#8 \\\/srv/owncloud\\\/lib\\\/base.php(874): OC\\\\Route\\\\Router->match('\\\/apps\\\/music\\\/api...')\\n#9 \\\/srv/owncloud\\\/index.php(40): OC::handleRequest()\\n#10 {main}\",\"File\":\"\\\/srv/owncloud\\\/apps\\\/music\\\/db\\\/track.php\",\"Line\":97}","level":4,"time":"2015-09-20T23:45:11+00:00"}

**Comment by Wikinaut** [comment]
OC 8.2.0  problem still exists. When i click onto an mp3 file, the music app shows this overview window(Why? I don't need this), but does not play the mp3.

What are you doing??

Can anyone use the Music app ?

**Comment by MorrisJobke** [comment]
This seems to be a bug ...the fileids seem to have changed. I will check how to make it more robust to cases like this.

**Comment by Wikinaut** [comment]
@MorrisJobke Why don't you implement a very easy player (plays, or try to play, ogg, mp3, wav) like in my gist https://gist.github.com/Wikinaut/d3a91158ef111c8c419b ? I can send you a link to page working with this by private mail. Contact me via please.

**Comment by Wikinaut** [comment]
Basically it is 

```
        $('#5').click(function() {
                $("audio").remove();
		var audio = $("<audio autoplay='autoplay'>");
        	addSource(audio, 'audio/eagle.ogg');
	        addSource(audio, 'audio/eagle.mp3');
	        addSource(audio, 'audio/eagle.wav');
        	$(this).after(audio).parent().addClass("ok");
        	$("audio").bind("ended",function(){$(this).remove()})
     	        	  .bind("canplay",function(){$(this).parent().removeClass().addClass("ok")})
		$("source").bind("error", function(){ alert("Error:\nSorry, cannot load or play this source file "+$(this).attr("src"));$(this).parent().remove() })
        });

        function addSource(elem, path) {
        	$('<source>').attr('src', path).appendTo(elem);
        }
```

**Comment by Wikinaut** [comment]
@MorrisJobke I just sent you a private mail with a kind request.

**Comment by paulijar** [comment]
The original problem of playback not starting and cursor jumping over songs has got very likely fixed with #535. The next mentioned problem of exception and inability to open the Music app is probably duplicate to #357 and should be no longer an issue. The request for "very easy player" without library view is fulfilled by #577 which is included in the brand new release v0.5.0.
